openapi.yaml

```diff
@@ -9804,6 +9782,85 @@ components:
           $ref: '#/components/headers/Cache-Control'
         Expires:
           $ref: '#/components/headers/Expires'
+    Workspace:
```

Comment:
Added a "Workspace" response object for creating, retrieving and updating a workspace endpoints response